client/src/Components/ApplicantCard/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FaHeart} from 'react-icons/fa';
+import { BiLike} from 'react-icons/bi';
+
+import './applicantcard.scss';
+
+const ApplicantCard = ({
+applicant, handleSuperLike, handlelike
+}) => {
+  return (
+    <>
+      <section className="applicantcard">
+        <div className="applicant">
+        <img className="applicant__avatar" src={`${applicant.image}`} alt="company logo" />
```

Comment:
src={applicant.image} should be enough, try that.

Reply:
resolved